paasta_tools/deployd/metrics.py

```diff
@@ -26,12 +23,9 @@ def __init__(self, inbox, cluster, metrics_provider):
     def run(self):
         while True:
             self.instances_to_bounce_later_gauge.set(
-                self.instances_to_bounce_later.qsize(),
-            )
-            self.instances_that_need_to_be_checked_up_on_gauge.set(
-                len(self.inbox.keys()),
+                self.queue.unavailable_service_instances.qsize(),
             )
             self.instances_to_bounce_now_gauge.set(
-                self.instances_to_bounce_now.qsize(),
+                self.queue.available_service_instances.qsize(),
             )
```

Comment:
Although not in this PR, with this change it would be neat to add additional metrics, like "oldest entry in the available queue" and "average bounce_by time" or something